packages/db-objects-file/lib/objects/objectsInMemFileDB.js

```diff
@@ -2006,21 +2006,11 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
             if (options.user) {
                 obj.acl.owner = options.user;
             }
-            // take the group as current user's group
+            // take the current user as owner if given
             if (options.group) {
                 obj.acl.ownerGroup = options.group;
-            } else  {
-                obj.acl.ownerGroup = null;
-                return this.getUserGroup(obj.acl.owner, (user, groups /* , permissions */) => {
```

Comment:
Why get the user group of the current owner when there is already a group hard defined in `system.config`. With this implementation the configured group will never be applied

Reply:
basically: how it is on linux FS? aahhhh there is no "system wide" things ;-)
So ...hm ... but yes when applying default it could be that the user can not access the oject he just created.
So maybe "use users rgroup if "not admin" and for admin use "system default"?

Reply:
I think the reason why it uses the users group is exactly that BF tried to mimic Linux FS permission management. When you create a file as a user you get "your" user group ... (and in fact root as "everything".

So the "issue" is that we have these "default rights" and the more I thik the more this should be applied for "admin actions" only

Reply:
Okay, I see two options, 

1. the on you proposed, so if user is admin apply default else apply given user/group
2. do not respect user/group attribute for setting acl but only to see if the user is allowed to call the setObject, so the only possibility to set acl would be to provide the `obj.acl` yourself, else default applies. This would be most clear structure but not that Linux FS like. 

Opinions @GermanBluefox @AlCalzone

Reply:
I don't understand 2. 😅

Reply:
Second is, `object.user` and `object.group` will not affect the `obj.acl` property of the created object. (But of course, they will be used to check if write access for objects is given for the user/group, else the setObject call will be rejected)